notebooks/notebooks/iclr2022/estimate-flops-adv.ipynb

```diff
@@ -23,8 +23,18 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        return game_info.read_and_parse_all_files(
</pre><p>tiny nit: we could just have this function return a pd.DataFrame?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Fixed.